include/aws/io/event_loop.h

```diff
@@ -115,6 +115,8 @@ struct aws_event_loop_group {
     struct aws_atomic_var current_index;
 };
 
+typedef void(aws_event_loop_group_cleanup_complete_fn)(void *user_data);
```

Comment:
we usually pass along the pointer to the event-loop-group

Reply:
In cases where it's not safe (the loop is gone) I think it's up to the user to create some sort of association via user data if they need to associate specific callbacks with specific loops.